be/src/vec/functions/function_java_udf.h

```diff
@@ -76,27 +76,37 @@ class JavaFunctionCall : public IFunctionBase {
     jmethodID executor_evaluate_id_;
     jmethodID executor_close_id_;
 
+    struct IntermediateState {
+        size_t buffer_size;
+        size_t row_idx;
+    };
+
     struct JniContext {
         JavaFunctionCall* parent = nullptr;
 
         jobject executor = nullptr;
 
         int64_t input_values_buffer_ptr;
         int64_t input_nulls_buffer_ptr;
-        int64_t input_byte_offsets_ptr;
+        int64_t input_offsets_ptrs;
```

Comment:
maybe use unique_ptr to manage the mem is better， the int64_t only use in thrift struct

Reply:
done